src/subcommand/server.rs

```diff
@@ -885,6 +909,36 @@ impl Server {
     Some((headers, inscription.into_body()?))
   }
 
+  async fn content_sat(
```

Comment:
@raphjaph this question is outside the route name. Do we only want to return the id from this endpoint? Should I take out this content endpoint that its also returning in this pr? 
`/content/sat/:sat/:page_index -> Content`